cmd/attack/redis.go

```diff
@@ -116,14 +117,33 @@ func NewRedisCacheLimitCommand(dep fx.Option, options *core.RedisCommand) *cobra
 		},
 	}
 
-	cmd.Flags().StringVarP(&options.Addr, "addr", "a", "", "")
+	cmd.Flags().StringVarP(&options.Addr, "addr", "a", "", "The address of redis server")
 	cmd.Flags().StringVarP(&options.Password, "password", "p", "", "The password of server")
 	cmd.Flags().StringVarP(&options.CacheSize, "size", "s", "0", "The size of cache")
 	cmd.Flags().StringVarP(&options.Percent, "percent", "", "", "The percentage of maxmemory")
 
 	return cmd
 }
 
+func NewRedisCacheExpirationCommand(dep fx.Option, options *core.RedisCommand) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "cache-expiration",
+		Short: "expire keys in Redis",
+		Run: func(*cobra.Command, []string) {
+			options.Action = core.RedisCacheExpirationAction
+			utils.FxNewAppWithoutLog(dep, fx.Invoke(redisAttackF)).Run()
+		},
+	}
+
+	cmd.Flags().StringVarP(&options.Addr, "addr", "a", "", "The address of redis server")
+	cmd.Flags().StringVarP(&options.Password, "password", "p", "", "The password of server")
+	cmd.Flags().StringVarP(&options.Key, "key", "k", "", "The key to be set a expiration, default expire all keys")
+	cmd.Flags().StringVarP(&options.Expiration, "expiration", "", "0", "The expiration of the key")
```

Comment:
Does this flag represent expiration time?  Can we add an example of this flag?

Reply:
Yes, and I'll enrich the description of `expiration`.